brandingbrand/flagship
# fix(flagship): usage description update

## Description
update of usage descriptions when used with react-native-permissions module was getting an error
when adding usage descriptions it doesn't add spaces
so for when we update look for 0 or more spaces in regex instead of 1 or more

## Test
* in a flagship project set environment values that contain below
```json
{
  "usageDescriptionIOS": [
      {
          "key": "NSLocationWhenInUseUsageDescription",
          "string": "initial value"
      }
  ],
  "permissions": {
    "ios": {
        "LOCATION_WHEN_IN_USE": "updated value",
    }
  }
}
```
* run flagship init with that environment
* check ios/[project]/Info.plist
* ER: NSLocationWhenInUseUsageDescription should show updated value

Additional Notes: having both values setup is probably not really good practice and would probably lead to subtle issues where you accidentally didn't update the correct value. So I would not recommend it, though it is a mistake that I made to find this.
